### ISSUE TITLE
State does not display on the right

### ISSUE THREAD

**Issue by nuttytree** [thomasloven/lovelace-template-entity-row#27]
I am seeing the same display issue on version .109.x and .110.x as #20 

![image](https://user-images.githubusercontent.com/22163869/84045746-cb44c580-a96e-11ea-9115-eadfe0e3a6a6.png)

Lovelace Config:
```
  - title: Test
    panel: false
    path: test
    cards:
      - title: Test
        type: entities
        entities:
          - entity: binary_sensor.patio_lights_status
          - type: 'custom:template-entity-row'
            entity: sensor.day_night
          - type: 'custom:template-entity-row'
            entity: binary_sensor.patio_lights_status
            name: >
              {{ state_attr(config.entity, 'friendly_name') | replace(' Status', '') }}
            active: >
              {{ is_state(config.entity, 'on') }}
            state: >
              {% if is_state(config.entity, 'on') %}Connected{% else %}Disconnected{% endif %}
            secondary: >
              {% set wifi = config.entity | replace('binary_sensor.', 'sensor.') | replace('_status', '_wifi_signal') %} {% set signal = states(wifi) %}
              {% if wifi != 'test' %}{{ signal }} dB{% endif %}
```

Browser console:
```
BROWSER_MOD 1.0.2 IS INSTALLED    DeviceID: c88dd5e1-14cdec8b
CARD-TOOLS IS INSTALLEDDeviceID: c88dd5e1-14cdec8b
CUSTOM-HEADER    Version 1.5.0
CALENDAR-CARD     Version 3.108.6
RGB Light Card  v1.6.0
LIGHT-ENTITY-CARD     Version 3.2.2
LAYOUT-CARD 1.2.1 IS INSTALLED
SLIDER-ENTITY-ROW 1.0.0 IS INSTALLED
TEMPLATE-ENTITY-ROW 1.0.1 IS INSTALLED
simple-thermostat: 0.34.2
SWPIE-CARD   Version 3.2.1
```

**Comment by richard-scott** [comment]
I’ve just had this start since the HA 2021.6.0 update. Removing this repo via HACS makes things normal again. Only happens on my iPhone tho.

**Comment by richard-scott** [comment]
The picture of my central heating items were not templates entity rows, they are normal entity items that get the formatting messed up when I install/activate template-entity-row.

The odd thing is that thus screenshot do not use this module and they are standard entities.

**Comment by thomasloven** [comment]
Fixed in release 1.1.2
